testgen/package.json

```diff
@@ -9,6 +9,7 @@
   "author": "",
   "license": "Unicode-3.0",
   "dependencies": {
+    "temporal": "^0.7.1",
```

Comment:
`temporal` is not the right package. You want `temporal-polyfill` as in the next line.

```suggestion
```